.github/workflows/publish.yml

```diff
@@ -108,3 +113,33 @@ jobs:
           else
             echo "No changes to commit"
           fi
+
+      - name: Verify the new package is available
+        run: |
+          published=none
+          ver_expected="${{ steps.sdk-ver.outputs.version }}"
```

Comment:
you put it in script. why the duplication ?

Reply:
The try.sh got in by mistake. I'll remove